game/dcs/aircrafttype.py

```diff
@@ -90,12 +90,35 @@ def make_namer(cls, config: dict[str, Any]) -> Type[ChannelNamer]:
         }[config.get("namer", "default")]
 
 
+@dataclass(frozen=True)
+class PatrolConfig:
+    altitude: Optional[Distance]
+    speed: Optional[Speed]
+
+    @classmethod
+    def from_data(cls, data: dict[str, Any]) -> PatrolConfig:
+        altitude = data.get("altitude", None)
+        speed = data.get("altitude", None)
+        return PatrolConfig(
+            feet(altitude) if altitude is not None else None,
+            knots(speed) if speed is not None else None,
+        )
+
+
 @dataclass(frozen=True)
 class AircraftType(UnitType[FlyingType]):
     carrier_capable: bool
     lha_capable: bool
     always_keeps_gun: bool
+
+    # If true, the aircraft does not use the guns as the last resort weapons, but as a main weapon.
+    # It'll RTB when it doesn't have gun ammo left.
+    gunfighter: bool
+
     max_group_size: int
+    patrol_altitude: Optional[Distance]
+    patrol_speed: Optional[Speed]
+    has_tacan: bool
```

Comment:
Looks like pydcs actually has this info: `FlyingType.tacan`.

Reply:
Oh, perfect. I've removed the new attribute and replaced it with this.